fundamental-concepts/choices-role-pattern/3_consuming_choices.md

```diff
@@ -0,0 +1,113 @@
+A choice can be `consuming` or `nonconsuming`.
+
+![choice_consuming](assets/choice_consuming.png)
+
+A `nonconsuming` choice doesn't affect the contract it's contained in. You can execute it as many
+times as you want. In contrast, a `consuming` choice can be called exactly once. As soon as the call
+happens, the containing contract is marked as inactive and a subsequent call will fail.
+
+Clearly, this is not what we want for an endpoint like
+
+```
+sendMessage: Existing User -> Party -> Text -> Update (Existing Message)
+```
+
+It would be really annoying if you would loose your account on the social network whenever you want
+to send a message to someone else. Hence, the corresponding choice `SendMessage` is declared as
+`nonconsuming`.
+
+But let's imagine we want an additional contract template to make special offers to other users in
+`create-daml-app`:
+
+<pre class="file" data-filename=`daml/User.daml` data-target="append">
```

Comment:
```suggestion
<pre class="file" data-filename="daml/User.daml" data-target="append">
```
This should work (single or double quotes). Otherwise it doesn't and you get an error:

[Extension Host] Opening Path /root/create-daml-app/`daml/User.daml`
console.ts:137 [Extension Host] [Katacoda] Unable to write file Error: ENOENT: no such file or directory, open '/root/create-daml-app/`daml/User.daml`'
log.ts:197   ERR ENOENT: no such file or directory, open '/root/create-daml-app/`daml/User.daml`': Error: ENOENT: no such file or directory, open '/root/create-daml-app/`daml/User.daml`'

Reply:
thanks, fixed.